maths/greater_common_divisor.py

```diff
@@ -1,25 +1,25 @@
-"""
-Greater Common Divisor.
+# Python program to find the G.C.D of two input number
 
-Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
```

Comment:
Why should we take this away??

Reply:
Sharing My Experience.When i was at my initial Learning stage i used to face several difficulties undersatnding code which contains complexities.So i prefer keeping codes ShortnSweet.